docs/section5/Rule5-1.md

```diff
@@ -20,44 +20,42 @@
 
-- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
 
-  - For each thermal_block in building segment: ```thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+  - For each thermal_block in building segment: ```thermal_block_b in building_segment_b.thermal_blocks:```  
   
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```  
+  - For each zone in thermal block: ```zone_b in thermal_block_b.zones:```  
 
-  - For each space in zone: ```space_baseline in zone_baseline.spaces:```  
+  - For each space in zone: ```space_b in zone_b.spaces:```  
 
-    - For each surface in space: ```for surface_baseline in space_baseline.surfaces:```  
+    - For each surface in space: ```for surface_b in space_b.surfaces:```  
 
-      - Check that surface is exterior and vertical: ```if ( surface_baseline.adjacent_to == "AMBIENT" ) AND ( 60 <= _surface.tilt <= 90 ): exterior_vertical_surface = surface_baseline```  
+      - Check that surface is exterior and vertical: ```if ( surface_b.adjacent_to == "EXTERIOR" ) AND ( 60 <= _surface.tilt <= 90 ): exterior_vertical_surface_b = surface_b```  
 
-      - Get the surface azimuth: ```surface_azimuth_baseline = surface_baseline.azimuth```  
+      - Get the surface azimuth: ```surface_azimuth_b = surface_b.azimuth```  
 
-      - Calculate the total fenestration area: ```fenestration_area_baseline = sum( fenestration.area for fenestration in exterior_vertical_surface.fenestration_subsurfaces)```  
-
-      - Get the wall area: ```wall_area_baseline = exterior_vertical_surface.area```  
+      - Calculate the total fenestration area: ```fenestration_area_b = sum( fenestration.area for fenestration in exterior_vertical_surface_b.fenestration_subsurfaces)```  
 
       - Summarize the total fenestration area and the total surface area for each orientation for the building:  
 
-      ```if ( surface_azimuth_baseline > 315 ) OR ( surface_azimuth_baseline <= 45 ): total_fenestration_area_north += fenestration_area_baseline```  
+      ```if ( surface_azimuth_b > 315 ) OR ( surface_azimuth_b <= 45 ): total_fenestration_area_north += fenestration_area_b```  
 
-      ```elsif ( surface_azimuth_baseline > 45 ) AND ( surface_azimuth <= 135 ): total_fenestration_area_east += fenestration_area_baseline```  
+      ```elsif ( surface_azimuth_b > 45 ) AND ( surface_azimuth_b <= 135 ): total_fenestration_area_east += fenestration_area_b```  
 
-      ```elsif ( surface_azimuth > 135 ) AND ( surface_azimuth <= 225 ): total_fenestration_area_south += fenestration_area_baseline```  
+      ```elsif ( surface_azimuth_b > 135 ) AND ( surface_azimuth_b <= 225 ): total_fenestration_area_south += fenestration_area_b```  
 
```

Comment:
@mkarpman and @JasonGlazer 90.1 is not very clear about what is north/south/east/west orienattion. Section 5.5.4.5 has the following-

- **East oriented**- oriented within 45 degrees of true east to the south and within 22.5 degrees of true east to the north in the northern hemisphere; oriented within 45 degrees of true east to the north and within 22.5 degrees of true east to the south in the southern hemisphere
- West Oriented- oriented within 45 degrees of true west to the south and within 22.5 degrees of true west to the north in the northern hemisphere; oriented within 45 degrees of true west to the north and within 22.5 degrees of true west to the south in the southern hemisphere
- **north-oriented** - facing within 45 degrees of true north in the northern hemisphere (however,
facing within 45 degrees of true south in the southern hemisphere).
- south oriented- there is no definition of this.

if N is within 45 deg of true north and East is within 22.5 degrees of ttrue east to the North, then what happens to the 22.5 degrees remaining to the North? What orientation is that. Same question for W. 

Let me know what I'm missing.

Reply:
@supriyagoel I don't see a definition for north oriented, where is that?

Reply:
@JasonGlazer the definition of north-oriented is actually in the 'definitions' section. I have edited my comment to clarify that

Reply:
In addition, Table G3.1 part 8 proposed exception 2 and Table G3.1 part 8 proposed item (b). Both of these assume a simple 45 binning of orientation. Although to me, the language implies four quadrants of 90 degrees each.

Reply:
This exception is that rotation is not required for "Buildings where the vertical fenestration area on each orientation varies by less than 5%." 

There is also the following in the definition section:  "orientation: the direction an envelope element faces, i.e., the direction of a vector perpendicular to and pointing away from the surface outside of the element."

So I wonder if, instead separating surfaces by exposure (N, NE, NW, etc.), we should just use azimuth as criteria for totation.

Reply:
That might be the simplest approach and it depends only on the defined term of "orientation"

Reply:
So the logic might be to just calculate the total fenestration area for each unique azimuth, find max and min per azimuth, and if they differ by more than 5% then the baseline must be rotated.

Reply:
excellent- makes sense. Thank you. Summarized for @xingcx below.

- we need not find the total fenestration area for N, S, E, W (see the reason explained in MK comemnt above)
- Instead, we find the total fenestration area for each unique azimuth within a project
- Identify the min and max fenestration area for each azimuth
- if the min/max differ by >5%, then the baseline must be rotated.